lib/api.js

```diff
@@ -15,6 +16,8 @@ const makeDir = require('make-dir');
 const ms = require('ms');
 const chunkd = require('chunkd');
 const Emittery = require('emittery');
+const minimist = require('minimist');
```

Comment:
Could we use https://www.npmjs.com/package/yargs-parser instead? It's already in our dependency tree as part of `meow`.

(Though it looks like so is `minimist`, because Node.js modules, but for less apparent reasons.)

Reply:
I choose minimist, because it has reverse package, maybe there is another package for it?

Reply:
I reckon `dargs` may work with `yargs-parser` too, assuming it also outputs straight-forward objects.